README.md

```diff
@@ -58,6 +58,24 @@ To do the initial setup and ensure that all tests pass locally:
 make
 ```
 
+#### Feature Tests
+
+Setup steps:
+
+1. Edit _.env_ file
+  * *API\_REDIRECT*=__http://web:8080__
+  * *API\_BASE\_URL*=__http://api:3000__
+1. Disable *SAML\_ENABLED* auth and enable *BASIC\_ENABLED*
+1. OPTIONAL Add _web_ and _api_ to the end of the localhost line in /etc/hosts so you don't have to keep changing the .env file
+
+To run the feature specs locally:
+
+```shell
+make specs
```

Comment:
I still get this error as I reported previously on the ticket, but I'm on RHEL7 which may be making the difference:
```
ERROR: Forbidden path outside the build context: ../conf/docker/Dockerfile.frontend ()
make: *** [specs] Error 1
```
I can chalk it up to platform differences and open up a separate ticket if you aren't seeing it on macOS.

Full error below:
```
[root@localhost eqipprototype]# make specs
Running integration test suite
Pulling hub (selenium/hub:latest)...
Trying to pull repository registry.access.redhat.com/selenium/hub ... 
Trying to pull repository docker.io/selenium/hub ... 
latest: Pulling from docker.io/selenium/hub
18d680d61657: Pull complete
0addb6fece63: Pull complete
78e58219b215: Pull complete
eb6959a66df2: Pull complete
f81eb8f33dd7: Pull complete
0db0c3aef5ed: Pull complete
956cc35674a3: Pull complete
8f06d7de8e79: Pull complete
ff4ef0987943: Pull complete
497ae96e252b: Pull complete
f0575dd5a9f9: Pull complete
5589e00cac87: Pull complete
745a29a8c5de: Pull complete
ece4310077aa: Pull complete
d856b41ebcd1: Pull complete
e0cb9064983b: Pull complete
Digest: sha256:8cebebfbdd6a49a262d5d7253021cfab19229655e770723a87cffb2a49cd6e60
Status: Downloaded newer image for docker.io/selenium/hub:latest
Pulling chrome (selenium/node-chrome:latest)...
Trying to pull repository registry.access.redhat.com/selenium/node-chrome ... 
Trying to pull repository docker.io/selenium/node-chrome ... 
latest: Pulling from docker.io/selenium/node-chrome
18d680d61657: Already exists
0addb6fece63: Already exists
78e58219b215: Already exists
eb6959a66df2: Already exists
f81eb8f33dd7: Already exists
0db0c3aef5ed: Already exists
956cc35674a3: Already exists
8f06d7de8e79: Already exists
ff4ef0987943: Already exists
497ae96e252b: Already exists
f0575dd5a9f9: Already exists
5589e00cac87: Already exists
745a29a8c5de: Already exists
75ffd7604bd6: Pull complete
3ac9a977c9b4: Pull complete
3313e71a9e00: Pull complete
2ff178eeb5bc: Pull complete
0d16bf8e6e17: Pull complete
91279c9d40b8: Pull complete
fb51dddbd7e6: Pull complete
ad45e081d6bc: Pull complete
cf4bb5b8efc0: Pull complete
d44c747e9294: Pull complete
4f1dd103af3d: Pull complete
3fb22b083722: Pull complete
Digest: sha256:6dc097413e4f7e5b8baf656145d66168025e9c697c0b0ac5e3d3f89f75fdc131
Status: Downloaded newer image for docker.io/selenium/node-chrome:latest
Creating eqipprototype_hub_1    ... done
Creating eqipprototype_hub_1    ... 
Creating eqipprototype_chrome_1 ... done
Building nightwatch
ERROR: Forbidden path outside the build context: ../conf/docker/Dockerfile.frontend ()
make: *** [specs] Error 1
```

Reply:
I am not seeing this on the Mac, but I'll see if I can find a better way around it.

Reply:
Docker for Mac is running 18.06.1-ce at the time of this writing, which includes the improvement to allow Dockerfile to exist outside of the build context which is what is resulting in this error. https://stackoverflow.com/questions/27068596/how-to-include-files-outside-of-dockers-build-context

As I recall RedHat only supports EE or docker version 1.13.1 or less, which is before the split of ce and ee versions. The bug was fixed in 18.03, which is probably causing the discrepancy between our setups. Please confirm if my suspicion is correct. If it is we need to upgrade docker on your REHL machine, or if that is not possible I can move the Dockerfile so that it will work. Let me know and I'll make the appropriate changes.

Reply:
I created a copy of the docker file for spec tests and that still runs on my box. Can you give it a go when you have a chance? Since this is late in the day I'll plan to move the story tomorrow if you don't have time to get to it today.